modules/nw-networkpolicy-about.adoc

```diff
@@ -12,7 +12,9 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [WARNING]
 ====
-Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules.
+Network policy does not apply to the host network namespace. Pods with host networking enabled are unaffected by network policy rules. However, if they are connected to pods with host network policies, they might be affected by the network policy rules.
```

Comment:
However, if they are connected to pods with host network policies, they might be affected by the network policy rules.

Rephrase it to. 
However pods connecting to the  host networked pods might be affected by the network policy rules.

Reply:
Ack.